sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -544,6 +549,14 @@ def test_piecewise_simplify():
     assert Piecewise((1, x < 1), (2, (x >= 2) & (x <= 3)), (3, True)
         ).simplify() == Piecewise((1, x < 1), (2, (x >= 2) & (x <= 3)),
         (3, True))
+@XFAIL
+def test_piecewise_simplify_xfail_redundant_inequality():
+    # Uncomment this case in test_piecewise_simplify() when it is fixed.
+    # See https://github.com/sympy/sympy/pull/25605
+    assert Piecewise((1, x < 2), (2, (x <= 3) & (x > 1)), (3, True)
+        ).simplify() == Piecewise((1, x < 2), (2, x <= 3), (3, True))
+
+
 
 
 def test_piecewise_solve():
```

Comment:
There should be two blank lines between top-level functions, classes etc (although some parts of the codebase use only one blank line, consistency within a file is the main thing). Here you have zero blank lines above and 4 blank lines below.